### PR TITLE
Add new tests for ensemble client

### LINKED ISSUE #1484: Improve ensemble client
Take a look at the exception handling and add tests for the exceptions

### PR DESCRIPTION
**Pull request by frode-aarstad**
Resolves #1484

### CODE REVIEW

File: tests/ensemble_evaluator/test_ensemble_client.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import threading
+import pytest
+import asyncio
+import websockets
+from functools import partial
+from ert_shared.ensemble_evaluator.prefect_ensemble.client import Client
+
+
+def _mock_ws(host, port, messages, delay_startup=0):
```
Comment (DanSava) [inline tests/ensemble_evaluator/test_ensemble_client.py:9]: Would it be possible to merge this function with the one from `test_prefect_ensemble` and move it to `tests/ensemble/evaluator/conftest.py`?

**Review by DanSava — COMMENTED** [review]
Looks good to me just one question.

File: tests/ensemble_evaluator/test_ensemble_client.py (hunk 1 above)
Comment (mortalisk) [reply tests/ensemble_evaluator/test_ensemble_client.py]: It would. Will it be done? That is another question, to which I can only reply, yes.

**Review by DanSava — APPROVED** [review]
LGTM!

**Pull request merged**